WORKSPACE

```diff
@@ -170,6 +170,20 @@ grpc_java_repositories(
     omit_net_zlib = True,
 )
 
+# Apply patches to protobuf to prevent duplicate symbols: https://github.com/lyft/envoy-mobile/issues/617.
+# TODO: Remove after https://github.com/bazelbuild/bazel/pull/10493 is merged to Bazel and we upgrade.
+http_archive(
```

Comment:
This has to be above envoy's setup otherwise the one that it loads will take precedence

Reply:
Updated